modules/treetime_validation.nf

```diff
@@ -2,7 +2,7 @@
 
 nextflow.enable.dsl = 2
 
-params.subtrees = Channel.of(20, 50, 100, 200, 500, 750, 1000, 1250, 1500, 2000)
+params.subtrees = Channel.of(20, 50, 100, 200, 500, 750, 1000)//, 1250, 1500, 2000)
```

Comment:
@4ment I only ran with the smaller parameter inputs. I can change this line to include the full range of parameters. And once we merge, then you can try running it yourself?

Reply:
Sounds good, please include full range